proto/p4/p4runtime.proto

```diff
@@ -136,6 +136,14 @@ message TableEntry {
   // this is stored, as long as it is returned with the rest of the entry in
   // a Read RPC.
   uint64 controller_metadata = 5;
+  // meter_config and counter_data are convenience fields that enable the
+  // controller to configure the direct resources associated with the table at
+  // the same time as a match-action entry is inserted or modified. When
```

Comment:
I wanted to capture initialization. Rephrased:

"If the table has a direct resource and a table entry is inserted without setting the direct resource field, the field values are initialized to 0."

While this seems reasonable for counters, I am hoping for a better solution for direct meters. Perhaps, we should return an error if meter config is not set, similar to action parameters not being set? Accidentally specifying a peak rate of 0 to a meter would turn the meter red by default, which may end up black-holing the traffic if the action is to rate-limit on meter red!